README.md

```
@@ -1,61 +1,137 @@
 > **Note**
 > This project uses volta, so please [set it up](https://docs.volta.sh/guide/getting-started). Also, read: [Why use Volta?](https://docs.volta.sh/guide/#why-volta)
 
+Ensure that you follow the TDD approach to writing your code.
+The tests are available and marked as `skip`. Enable your tests by removing the `.skip` from test or describe.
+
+Ensure that your milestone tests are passing before submitting the PR
+
+
+## 📋 Project Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)